nephos/fabric/utils.py

```diff
@@ -105,6 +105,83 @@ def get_helm_pod(namespace, release, app, item=0):
     return get_pod(namespace, identifier, item=item)
 
 
+def get_org_tls_ca_cert(opts, msp_namespace):
+    """Get path to the directory containing tls CA certificate
+
+        Args:
+            opts (dict): Nephos options dict.
+            msp_name (str): Name of Membership Service Provider.
+
+        Returns:
+            path: path to the directory containing tls CA certificate
+    """
+
+    if "tls_ca" in opts["ordering"]["tls"]:
+        glob_target = f"{opts['core']['dir_crypto']}/tlscacerts/*.crt"
+    else:
+        glob_target = f"{opts['core']['dir_crypto']}/crypto-config/*Organizations/{msp_namespace}*/tlsca/*.pem"
+    tls_path_list = glob(glob_target)
+    if len(tls_path_list) == 1:
+        return tls_path_list[0]
+    else:
+        raise ValueError(
+            f"TLS path list length is {len(tls_path_list)} - {tls_path_list}"
+        )
+
+
+def get_tls_path(opts, id_type, namespace, release):
+    """Get path to the directory containing tls materials for a node
+        Args:
+            opts (dict): Nephos options dict.
+            id_type (str): Type of ID we use.
+            namespace (str): Name of namespace.
+            release (str): Name of release/node.
+
+        Returns:
+            path: path to the directory containing materials of a node
```

Comment:
"str: Path ..."

Return arguments should declare the type of the argument, rather than its name. See:

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html